src/avram/save_operation.cr

```diff
@@ -30,6 +32,7 @@ abstract class Avram::SaveOperation(T) < Avram::Operation
     @@schema_class = T
   end
 
+  property record
```

Comment:
Why is this exposed?

Reply:
So that it can be set in the upsert if it finds an existing record

Reply:
I see it now.